OpenTabletDriver/InputDevice.cs

```diff
@@ -9,6 +10,9 @@ namespace OpenTabletDriver
 {
     public class InputDevice : DeviceReader<IDeviceReport>
     {
+        private uint FeatureInitDelayMs;
+        private readonly string DelayAttributeKeyName = "FeatureInitDelayMs";
```

Comment:
Don't directly apply this suggestion, refactor from within your IDE.
```suggestion
        private uint _featureInitDelayMs;
        private const string DELAY_ATTRIBUTE_KEY_NAME = "FeatureInitDelayMs";
```